README.md

```diff
@@ -1,86 +1,71 @@
+![](https://github.com/nomad-coe/nomad-parser-nexus/actions/workflows/pylint.yml/badge.svg?event=push)
+![](https://img.shields.io/badge/python-3.8-green.svg)
 
-First, it offers users a tool with which they can
-create specific data files packaging together numerical data and metadata from
-experiments which can originate from technology partners and/or text file output
-from an electronic lab notebook (ELN).
+<br/>
 
-Second, the software is used in the German National Research Data Infrastructure
-(German NFDI) project FAIRmat, here specifically in the NOMAD OASIS research data
-management (RDM) system, to serve as a set of parsers which enable users of a NOMAD
-Oasis instance to create data and metadata entries from experiments so that NOMAD Oasis
-understands the specific logic and terminology of the scientific field.
+This is a software for developing ontologies and instances of ontologies with
+[NeXus](https://www.nexusformat.org/).
 
-The software tools are located inside the module subdirectory 'nexusutils'and they are
-shipped with unit tests located in the subdirectory 'tests'.
-Some examples with real datasets are also provided in the subdirectory 'examples'
-in which Jupyter Notebooks are guiding through the process of converting instrument raw
-data into the NeXus standard and then aloow the visualisation of the hierarchical contents
-of the generated NeXus file in JupyterLab.
+# Installation
 
-## Getting started
+It is recommended to use Python version 3.8 with this package. You can learn how to manage Python versions [here](https://github.com/pyenv/pyenv).
 
-You should create a virtual environment. We tested on Ubuntu with Python 3.7 and Python 3.8.
-We recommend to use Python 3.8 but acknowledge that NOMAD uses still Python 3.7.
+It is also recommended to use a fresh virtual environment to install all dependencies. You can learn more [here](https://realpython.com/python-virtual-environments-a-primer/).
 
-If you don't have Python 3.8 installed on your computer, follow these commands:
+Install this package:
 
-```
-sudo add-apt-repository ppa:deadsnakes/ppa
-sudo apt install python3.8 python3-dev libpython3.8-dev python3.8-venv
+```console
+pip install git+https://github.com/nomad-coe/nomad-parser-nexus.git
 ```
 
-In some cases when using Python3.7 it was necessary to install python-numpy in others
-not. So consider this if you run into issues when continuing this manual.
+<br/>
 
-You can now install your virtual environment with a python3.8 interpreter
+# Individual modules of this software and their functions
 
-```
-mkdir <your-brand-new-folder>
-cd <your-brand-new-folder>
-pip install virtualenv
-virtualenv --python=python3.8 .py38
-source .py38/bin/activate
-pip install pip-tools
-```
+### **nyaml2nxdl**
+
+This is a tool which converts, easy to read, YAML NeXus schemas into NXDL NeXus schemas. [NeXus Schemas](https://nexusformat.org)
+
+Further documentation: [README.md](nexusutils/nyaml2nxdl/README.md)
```

Comment:
Could we not just link the headers? This repeated `Further documentation: README` seems a bit redundant to me.

Reply:
Eventually, it may be best to have a readthedocs page for this? There we could also compile and display our jupyter examples with sphinx-gallery (Nothing we should do rn, because I don't think we can finish this until dpg)